src/vs/workbench/contrib/scm/browser/scm.contribution.ts

```diff
@@ -198,6 +198,13 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
 			markdownDescription: localize('inputFontFamily', "Controls the font for the input message. Use `default` for the workbench user interface font family, `editor` for the `#editor.fontFamily#`'s value, or a custom font family."),
 			default: 'default'
 		},
+		'scm.inputFontSize': {
+			type: 'number',
+			minimum: 6,
+			maximum: 100,
+			markdownDescription: localize('inputFontSize', "Controls the font size for the input message in pixels."),
+			default: 13
```

Comment:
I took the default value from the previous [configuration](https://github.com/microsoft/vscode/blob/480b74c02e59ad9103a5598df0ec39d9e010b875/src/vs/workbench/contrib/scm/browser/scmViewPane.ts#L1480).